packages/generator/ni_measurement_plugin_sdk_generator/client/templates/measurement_plugin_client.py.mako

```diff
@@ -111,17 +115,43 @@ class ${class_name}:
         self._pin_map_client = pin_map_client
         self._stub: Optional[v2_measurement_service_pb2_grpc.MeasurementServiceStub] = None
         self._measure_response: Optional[
-            Generator[v2_measurement_service_pb2.MeasureResponse, None, None]
+            grpc.CallIterator[v2_measurement_service_pb2.MeasureResponse]
         ] = None
-        self._configuration_metadata = ${configuration_metadata}
-        self._output_metadata = ${output_metadata}
+        self._configuration_metadata = {
+            % for key, value in configuration_metadata.items():
+            ${key}: ParameterMetadata(
+                display_name="${value.display_name}",
+                type=Field.Kind.ValueType(${value.type}),
+                repeated=${value.repeated},
+                default_value=${_format_default_value(value.default_value)},
+                annotations=${value.annotations},
```

Comment:
This uses the default filter of `str`. Try using `${value.annotations | repr}` (or `${value.annotations | n,repr}`) to be more explicit about it.

Reply:
Sure, I have updated with `${value.annotations | n,repr}`.